MANIFEST.in

```diff
@@ -1,9 +1,6 @@
-include aiida/backends/tests/export_import_test_files/*.aiida
-include aiida/backends/sqlalchemy/migrations/script.py.mako
 include aiida/cmdline/templates/*.tpl
 include aiida/manage/backup/backup_info.json.tmpl
-include docs/source/images/*.png
```

Comment:
I think this was there because it is used by the `README.md` but I see that that is not included. Should it be? Is it common practice to ship the readme with a package?

Reply:
Well, we actually _require_ the `README.md` to be there [because it's read in `setup.py`](https://github.com/aiidateam/aiida-core/blob/develop/setup.py#L49) - I think it's included automatically. 

Doesn't mean the images necessarily need to be included though.

Complete side note: Notice how we're keeping the `README.md` file open throughout the entire setup process 😅

Reply:
There's actually another issue with the REAdme.md - would you mind replacing the relative links with absolute ones?

Otherwise they obviously don't work on pypi: https://pypi.org/project/aiida-core/

Reply:
sorry to keep piling onto this but since the release is coming up, now would be the time... ;-)

Reply:
To clarify on my previous point (just checked this): The `README.md` is included in the `sdist` but removed in the `bdist_wheel` - which I think is what we want, since the wheels don't need to execute `setup.py`.

Reply:
@ltalirz absolute to what, though? As a URL?

Reply:
https://raw.githubusercontent.com/aiidateam/aiida-core/develop/docs/source/images/logo_aiida_main.png ? Bit ugly having to include the branch-name in the README, no?

Reply:
maybe we can make a permalink available through `aiida.net` and link that? That would be better already, no?

Reply:
Much better, yeah. That's also how `matplotlib` seems to do it.

Reply:
Let's use this link: http://www.aiida.net/wp-content/uploads/2020/06/logo_aiida.png
and then we can remove the `docs/**` line from the `MANIFEST.in`

Reply:
A `grep -inR logo_aiida_main .` comes up empty for me - should we remove the file also?

Reply:
Also, you may want to test this on TestPyPI to see if it actually works now..

Reply:
> A `grep -inR logo_aiida_main .` comes up empty for me - should we remove the file also?

It was being used in the documentation, but that was recently replaced by a documentation specific logo. 

> Also, you may want to test this on TestPyPI to see if it actually works now..

I have never done this, but it seems you need a separate user account. So I will see if I can create one and test your branch

Reply:
> > A `grep -inR logo_aiida_main .` comes up empty for me - should we remove the file also?
> 
> It was being used in the documentation, but that was recently replaced by a documentation specific logo. OK to remove then I would say

Done!

> 
> > Also, you may want to test this on TestPyPI to see if it actually works now..
> 
> I have never done this, but it seems you need a separate user account. So I will see if I can create one and test your branch

Yeah, it's an entirely separate PyPI instance - with the understanding that nothing that's on there should be used in any sort of production environment.